ONSdigital/takeon-ui
# change heading for tabs

# Description
Changed headings for tabs 
Story link: https://collaborate2.ons.gov.uk/jira/browse/SPP-3815

# How to test?
Use env 241 and see that the headings have been changed to: current response and historical responses

Feel free to check BDD tests have not broken `behave features/test_survey/historical_responses.feature features/test_survey/historical_responses_adjusted.feature -D browser=headless`
